analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -580,15 +581,27 @@ class Pub(
         return yamlMapper.readTree(definitionFile)
     }
 
+    override fun getVersion(workingDir: File?): String {
+        val result = ProcessCapture(workingDir, command(workingDir), getVersionArguments())
+        if (result.isError) {
+            throw IOException(result.errorMessage)
+        }
+
+        return transformVersion(result.stderr)
+    }
```

Comment:
From a quick glance this looks pretty much like the default implementation, so why is this overridden?